### PR TITLE
Enh/logo

### PR DESCRIPTION
**Pull request by JonathonMSmith**
# Description

Addresses #20 

added new logo png, and changed the README to reference the image locally

# Type of change

Please delete options that are not relevant.

- Aesthetics

# How Has This Been Tested?

I only checked that the image displayed properly on the github page for my fork

## Test Configuration
* Operating system: MacOS 10.15.7
* Version number: Python 3.8

# Checklist:

- [X] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [X] Add a note to ``CHANGELOG.md``, summarizing the changes

If this is a release PR, replace the first item of the above checklist with the
release checklist on the pysat wiki:
https://github.com/pysat/pysat/wiki/Checklist-for-Release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div align="left">
         <img height="0" width="0px">
-        <img width="20%" src="https://raw.githubusercontent.com/pysat/pysatSpaceWeather/main/docs/figures/pysatSpaceWeather.png" alt="pysatSpaceWeather" title="pysatSpaceWeather" </img>
+        <img width="20%" src="/docs/figures/pysatSpaceWeather.png" alt="pysatSpaceWeather" title="pysatSpaceWeather" </img>
```
Comment (aburrell) [inline README.md:3]: So, this change needs to be reverted.  It's needed for the logo to show up on PyPi.

**Review by aburrell — CHANGES_REQUESTED** [review]
Thanks for the lovely new logo!  Besides reverting the change in the README, you need to:

- [ ] Add your name to the .zenodo.json

File: README.md (hunk 1 above)
Comment (JonathonMSmith) [reply README.md:3]: I figured there was a reason.

**Review by aburrell — DISMISSED** [review]

File: .zenodo.json [hunk 2]
```diff
@@ -22,6 +22,11 @@
       {
         "name": "Spence, Carey",
         "orcid": "0000-0001-8340-5625"
-      }
+      },
+      {
+        "affiliation": "Goddard Space Flight Center", 
+        "name": "Smith, Jonathon M.",
+        "orcid": "0000-0002-8191-4765"  
+      },
```
Comment (aburrell) [inline .zenodo.json:30]:
```suggestion
      }
```

**Review by aburrell — COMMENTED** [review]
Apparently jsons don't like trailing commas.

**Pull request merged**